bin/benchmark_performance.rb

```diff
@@ -0,0 +1,425 @@
+require 'tmpdir'
+require 'memory_profiler' # MemoryProfiler does not work for JRuby
```

Comment:
CI has jruby 9.2. Should it not if this does not work?

Reply:
We still want to collect timing metrics for JRuby